images/regen-sandbox/setup/bridge.sh

```diff
@@ -0,0 +1,104 @@
+source $(dirname $0)/utils.sh
+
+set -eo pipefail
+
+TX_FLAGS="--from $ADDR1 --yes --fees 5000uregen"
+
+echo "INFO: Creating Credit Class - C01"
```

Comment:
As mentioned above, these ends up being inaccurate depending on the combination of scripts. I think we should make this more generic here and throughout the setup scripts without referencing the specific ids.